src/classes/PSC_PartialSoftCredit_TEST.cls

```diff
@@ -123,19 +123,13 @@ public with sharing class PSC_PartialSoftCredit_TEST {
         insert listPSC;
         
         // make sure new OCRs created
-        set<ID> setOCRId = new set<ID>();
-        for (OpportunityContactRole ocr : [select Id, Role from OpportunityContactRole]) {
+        listOCR = getOpportunityContactRoles();
+        for (OpportunityContactRole ocr : listOCR) {
             system.assertEquals('Soft Credit', ocr.Role);
-            setOCRId.add(ocr.Id);
         }
-        system.assertEquals(cOppTest, setOCRId.size());
+        system.assertEquals(cOppTest, listOCR.size());
         
-        // make sure OCR id's stored in PSCs
-        listPSC = [select Id, Contact_Role_ID__c from Partial_Soft_Credit__c];
-        system.assertEquals(cOppTest, listPSC.size());
-        for (Partial_Soft_Credit__c psc : listPSC) {
-            system.assertEquals(true, setOCRId.contains(psc.Contact_Role_ID__c));
-        }        
+        assertOpportunityContactRolesAreStoredInPartialSoftCredits(listOCR); 
```

Comment:
@njjc  Other test methods used Sets to verify OpportunityContactRole.Ids are stored in PSC, thus they are deterministic already. However, these tests were refactored to be consistent and to use the same data retrieval and verification methods throughout the test class. If there is any concern to making all unit tests uniform, I can easily revert back to the previous commit.